martbuilder_templates_xml/ensembl_template.xml

```diff
@@ -4455,8 +4455,8 @@
 		<renamedColumn tableKey="attrib_type" colKey="1.6.0.description" newName="description_106"/>
 		<renamedColumn tableKey="attrib_type" colKey="1.6.0.code" newName="code_106"/>
 		<renamedColumn tableKey="attrib_type" colKey="1.6.0.attrib_type_id" newName="attrib_type_id_106_key"/>
-		<restrictedTable tableKey="attrib_type" tableId="483" aliasColumnIds="484" aliasNames="col1" expression=":col1 IN (&quot;appris&quot;,&quot;TSL&quot;,&quot;gencode_basic&quot;,&quot;MANE_Select&quot;)"/>
-	</dataset>
+		<restrictedTable tableKey="attrib_type" tableId="483" aliasColumnIds="484" aliasNames="col1" expression=":col1 IN (&quot;appris&quot;,&quot;TSL&quot;,&quot;gencode_basic&quot;,&quot;MANE_Select&quot;,&quot;MANE_Plus&quot;)"/>
```

Comment:
The attrib_type.code is "MANE_Plus_Clinical", rather than "MANE_Plus".

Reply:
mchakiachvili@noah-login-01 ~ $ mysql-ens-sta-1 homo_sapiens_core_102_38 -e "SELECT * from attrib_type where code like 'MANE%'";
attrib_type_id	code	name	description
550	MANE_Plus	MANE Plus v0.91	MANE Plus (v0.91) is the preliminary release (phase 6) of the MANE Plus data set. The Matched Annotation from NCBI and EMBL-EBI project (MANE) is a collaboration between Ensembl-GENCODE and RefSeq to select a default transcript per human protein coding locus that is representative of biology, well-supported, expressed and conserved. This transcript set matches GRCh38 and is 100% identical between RefSeq and Ensembl-GENCODE for 5' UTR, CDS, splicing and 3' UTR.
535	MANE_Select	MANE Select v0.91	MANE Select (v0.91) is the preliminary release (phase 6) of the MANE Select data set. The Matched Annotation from NCBI and EMBL-EBI project (MANE) is a collaboration between Ensembl-GENCODE and RefSeq to select a default transcript per human protein coding locus that is representative of biology, well-supported, expressed and conserved. This transcript set matches GRCh38 and is 100% identical between RefSeq and Ensembl-GENCODE for 5' UTR, CDS, splicing and 3' UTR.

I thought the value used was the actual value from attrib_type code?

Reply:
which matches with related PR (id=550)

Reply:
This is for 103, not 102, so need to look at the db on st1b. That said, the attrib_type table in that core db has not been synced with the production db since the name was changed: http://production-services.ensembl.org/admin/ensembl_production_db/masterattribtype/?q=MANE